dev/ci/docker/README.md

```diff
@@ -1,37 +1,13 @@
 ## Overall Docker Setup for Coq's CI.
```

Comment:
We should say somewhere "if you change the dockerfile you need to change the cachekey in gitlab and circle yml files"

Reply:
I think now things are simple enough that even a commit check could work to ensure this.

Reply:
That would be great!